### PR TITLE
Dust off, update to Go 1.14, enable modules

### PR DESCRIPTION
**Pull request by rafecolton**

### CODE REVIEW

**Comment by rafecolton** [comment]
@jszwedko I enabled branch protection for the master branch, so it now requires:
* Signed commits
* At least one review from one of us (**edit:** or Dan)
* All tests must pass

Awaiting your review 😄

File: Bobfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (jszwedko) [inline Bobfile:1]: I had forgotten about this tool :smile:

**Review by jszwedko — APPROVED** [review]
Looks great! Thanks for brushing it off!

File: Bobfile (hunk 1 above)
Comment (rafecolton) [reply Bobfile:1]: Heh, I haven't actually updated it in years, but every time I go back to anything docker, I reach straight for docker-builder.

**Pull request merged**